zds/forum/search_indexes.py

```diff
@@ -17,9 +17,14 @@ class TopicIndex(indexes.SearchIndex, indexes.Indexable):
     author = indexes.CharField(model_attr='author')
     pubdate = indexes.DateTimeField(model_attr='pubdate')
 
+    tags = indexes.MultiValueField()
+
     def get_model(self):
         return Topic
 
+    def prepare_tags(self, obj):
+        return [tag.title for tag in obj.tags.all()]
```

Comment:
Ça marcherai aussi bien. Je trouve ça plus clair avec le for, à relire mais c'est aux gouts et couleurs la.